deconnable_everything.md

```diff
@@ -0,0 +1,107 @@
+  in question. More rarer and exotic objects like big scrubbers should obviously be harder to deconstruct, but rewarding more valuable mats in return.
+  
+  The primary goal here is to have the majority of *unintentionally* destructable objects be deconstructable.
+  
+  Giving more opportunities for engies to loot colony side will theorectically promote them salvage more and allow for them to have a little bit more 
+  independence from req by trading some element of risk. Potential RP opportunities are a plus side.
+
+  *ex: FOB runs out of mats and REQ isn't responding. Bravo SL takes the initiative and leads a salavaging team to colony atmos to deconstruct some machinery.*
+  
+3. Give some of the more used objects actual recipes to build them back again
+
+  More or less self explanitory. We dont want people's experience shafted because a cheeky MT deconstructed their fridge or hydroponics tray. Used objects should be
+  able to be built back at a reasonable cost.
+  
+4. Give crafting recipes to stuff that SHOULD have it but do not
+
+   There's still a few dumb things that should need crafting recipes like wooden tables and stuff like that. Stuff thats borderline unobtainable for 0 reason at all 
+   should be made deconstructable. Decor objects should obviously have recipes for RP.
+   
+   
+With all goals done it should give CM:
+  - Limited potential balance ramifications
+     - More freedom of movement would cause this.
+     - Mats would be more avalible colonyside. This wouldn't be too large since you trade risk for reward (lurkers) but its still a thing.
+  - Massive QoL for xenos and marines alike.
+    - Trust me, it may not seem like much, but this would change how a lot of us play because we instictually avoid snowflake objs.
+  - More engineer salvaging opporunities and req independence.
+  - Some more RP opporunities generated from decor objects being accessible.
+
+   
+
+<!-- Now's where you get into clear detail about everything your feature does. **You should still be explaining 'why' things are that way, *as* you describe what.** Be as detailed as possible. -->
+
+## Alternatives
+
+<!-- Provide potential alternatives to your feature, either ones that align with your design values, or ones that don't that you suspect will be suggested. If you are including the latter, make sure to explain why you didn't choose that. -->
+
+## Potential Changes
+
+- Reward bias being given toward objs found colonyside
+  - This could be done to promote more risky,but rewarding salvaging colonyside rather than COs bitching at MTs to deconstruct the entire ship every round.
+<!-- Most of the time you're not going to get the best design first try. It helps to try your best to predict what *could* go wrong, and suggest alternatives that can be taken, without sacrificing your design. -->
```

Comment:
Not what the potential changes section is for. Remove this section if you don't have any potential changes that may need to be done.
This section is about predicting what could go wrong with your feature, e.g. xenos deconstructing everything, and to suggest alternatives that still align with your design.